plasmapy/formulary/paschen.py

```diff
@@ -0,0 +1,279 @@
+     {'A': 11, 'B': 135, 'gam': 0.01}
+
+     If ``gas`` is not found, `None` is returned.
+
+     >>> get_paschen_constants("Zz", "Ni")
+
+
+ """
+
+    # Supported gases
+
+    gases = ["Air", "N2", "H2", "He", "Ne", "Ar", "Kr", "Xe"]
+
+    paschen_constants = {
+        "Air": [11, 274],
+        "N2": [9.0, 257],
+        "H2": [3.8, 104],
+        "N2": [9.0, 257],
+        "He": [2.3, 26],
+        "Ne": [3.0, 75],
+        "Ar": [11, 135],
+        "Kr": [13, 180],
+        "Xe": [20, 263],
+    }
+
+    # Supported electrode materials
+
+    materials = ["Al", "Cu", "Ni", "Pt", "C", "W", "Fe"]
+
+    townsend_gamma = {
+        "Air": {
+            "Al": 0.035,
+            "Cu": 0.025,
+            "Ni": 0.036,
+            "Pt": 0.017,
+            "C": None,
+            "W": None,
+            "Fe": 0.02,
+        },
+        "N2": {
+            "Al": 0.1,
+            "Cu": 0.066,
+            "Ni": 0.077,
+            "Pt": 0.59,
+            "C": None,
+            "W": None,
+            "Fe": 0.059,
+        },
+        "H2": {
+            "Al": 0.095,
+            "Cu": 0.05,
+            "Ni": 0.053,
+            "Pt": 0.02,
+            "C": 0.014,
+            "W": None,
+            "Fe": 0.061,
+        },
+        "He": {
+            "Al": 0.021,
+            "Cu": None,
+            "Ni": 0.015,
+            "Pt": 0.01,
+            "C": None,
+            "W": None,
+            "Fe": 0.015,
+        },
+        "Ne": {
+            "Al": 0.053,
+            "Cu": 0.02,
+            "Ni": 0.031,
+            "Pt": 0.023,
+            "C": None,
+            "W": 0.045,
+            "Fe": 0.022,
+        },
+        "Ar": {
+            "Al": 0.12,
+            "Cu": 0.058,
+            "Ni": 0.058,
+            "Pt": 0.058,
+            "C": None,
+            "W": None,
+            "Fe": 0.058,
+        },
+        "Kr": {
+            "Al": None,
+            "Cu": None,
+            "Ni": None,
+            "Pt": None,
+            "C": None,
+            "W": None,
+            "Fe": None,
+        },
+        "Xe": {
+            "Al": None,
+            "Cu": None,
+            "Ni": None,
+            "Pt": None,
+            "C": None,
+            "W": None,
+            "Fe": None,
+        },
+    }
+
+    # Check if the asked gas and electrode material is supported
+    resg = gas in gases
+    rese = electrode in materials
+
+    # If the gas is supported get the constants A and B
+    if resg:
+        A = paschen_constants[gas][0]
+        B = paschen_constants[gas][1]
+
+        #   Get the townsend_gamma coefficient for the the gas/electrode combination
+        if rese:
+            gam = townsend_gamma[gas]
+            gn = gam[electrode]
+            #       Test if townsend_gamma exists for the demanded gas/electrode configuration
+            #       If not a default townsend_gamma value of 0.01 is taken
+
+            if gn is None:
+                gn = 0.01
```

Comment:
I have literally no expertise with this area. Is 0.01 a sensible value "in general"? Perhaps, rather than assume it, we could raise a `KeyError: no data for the given gas/electrode`, or at the very least throw a warning and say that we're assuming the default. I think I'd prefer the former approach.